src/js/constants/selectors-map.ts

```diff
@@ -105,6 +105,7 @@ export const desktopMenu = {
 
 export const qtyInput = {
   default: '.js-quantity-button',
+  productCartList: '.cart__items .js-quantity-button',
```

Comment:
Looks like this is not a good place

It contains selectors of `qtyInput` parts, not where `qtyInput` is placed

maybe inside `cart` const? `cart.productQuantity`